.travis.yml

```diff
@@ -3,13 +3,10 @@ language: php
 sudo: false
 
 php:
-  - 7.0
```

Comment:
According to http://php.net/supported-versions.php, PHP 7.0 is still maintained version and will get security fixes up to December, 2018.

IMO it's too early to drop it. Currently, there are no PHP 7.1 features that we can't live without. We will just lose many users who are still on PHP7.0.x.

Please revert all the code related to PHP7.1